tests/UI/Component/Input/Field/TextAreaTest.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+/* Copyright (c) 2018 Jesús López <lopez@leifos.com> Extended GPL, see docs/LICENSE */
+
+require_once(__DIR__ . "/../../../../../libs/composer/vendor/autoload.php");
+require_once(__DIR__ . "/../../../Base.php");
+require_once(__DIR__ . "/InputTest.php");
+
+use ILIAS\UI\Implementation\Component\SignalGenerator;
+use \ILIAS\UI\Component\Input\Field;
+
+class TextAreaTest extends ILIAS_UI_TestBase {
```

Comment:
Please add tests for the getters and mutators of the `TextArea`-class and `isLimited` specifically.